klister.cabal

```diff
@@ -0,0 +1,120 @@
+cabal-version: 2.2
+
+name:           klister
+version:        0.1
+homepage:       https://github.com/gelisam/klister#readme
+bug-reports:    https://github.com/gelisam/klister/issues
+author:         David Christiansen <david@davidchristiansen.dk>,  Samuel Gélineau <gelisam@gmail.com>
+maintainer:     David Christiansen <david@davidchristiansen.dk>,  Samuel Gélineau <gelisam@gmail.com>
+license:        BSD-3-Clause
+license-file:   LICENSE
+tested-with:    GHC==8.6.5, GHC==8.8.4, GHC==8.10.2
+build-type:     Simple
+data-files:
+    stdlib/builtins.kl
+    stdlib/defun.kl
+    stdlib/n-ary-app.kl
+    stdlib/prelude.kl
+
+source-repository head
+  type: git
+  location: https://github.com/gelisam/klister
+
+common deps
+  build-depends:
+    base                 >= 4.12.0 && < 4.15,
+    bifunctors           >= 5.5.5 && < 5.6,
+    containers           >= 0.6 && < 0.7,
+    directory            >= 1.3.3 && < 1.4,
+    extra                >= 1.6.18 && < 1.8,
+    filepath             >= 1.4.2 && < 1.5,
+    lens                 >= 4.17.1 && < 4.20,
+    megaparsec           >= 7.0.5 && < 9.1,
+    mtl                  >= 2.2.2 && < 2.3,
+    prettyprinter        >= 1.2 && < 1.8,
+    text                 ^>= 1.2,
+    transformers         ^>= 0.5
+
+library
+  import: deps
+  exposed-modules:
+      Alpha
```

Comment:
Indentation here is 4 but in `common deps`/`build-depends:` it's 2. Any reason for the discrepancy?